axlearn/open_api/metrics/tool_use_execution.py

```diff
@@ -232,6 +249,165 @@ def _compare_tool_calls(
     return True
 
 
+@dataclasses.dataclass
+class DetailedMatchResult:
```

Comment:
If this is internal, consider marking it as private:
```suggestion
class _DetailedMatchResult:
```

Reply:
Done